src/vs/workbench/test/workbenchTestServices.ts

```diff
@@ -738,6 +738,18 @@ export class TestFileService implements IFileService {
 		});
 	}
 
+	resolveStringStream(resource: URI, options?: IResolveContentOptions): IStringStream {
```

Comment:
We need to ensure that the new code is covered by the same test suite that covers `resolveContent` today. There are many tests in [`fileService.test.ts`](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/services/files/test/node/fileService.test.ts#L628) that cover various things (e.g. large file, not modified since, encoding).